SECURITY.md

```diff
@@ -395,7 +395,8 @@ grpc-netty version | netty-handler version | netty-tcnative-boringssl-static ver
 1.18.x-1.19.x      | 4.1.32.Final          | 2.0.20.Final
 1.20.x-1.21.x      | 4.1.34.Final          | 2.0.22.Final
 1.22.x             | 4.1.35.Final          | 2.0.25.Final
-1.23.x-            | 4.1.38.Final          | 2.0.25.Final
+1.23.x             | 4.1.38.Final          | 2.0.25.Final
+1.24.x-            | 4.1.41.Final          | 2.0.25.Final
```

Comment:
This PR did not got into the 1.24 release, therefore here still need to be 4.1.38.Final.